src/compile/compile.ts

```diff
@@ -57,6 +57,26 @@ function scene(config) {
   }, {});
 }
 
+export function compileTitle(model: Model) {
+  const spec = model.spec();
+  return {
+      'name': 'title',
```

Comment:
Can you explain more on this? Use the method in model instead of giving `name: 'title` ?